scripts/transformationsv2_config_translations.py

```diff
@@ -1,36 +1,71 @@
+import hashlib
+import json
+
+import numpy as np
+import pandas as pd
+
 from pathlib import Path
 
 import yaml
 
 from cognite.powerops.resync.config import ReSyncConfig, Transformation
 from cognite.powerops.resync.models._shared_v1_v2.cogshop_model import transformations_v2_transformer
 
-REPO_ROOT = Path(__file__).resolve().parent.parent
+
+def immutify_dictionary(d):
+    d_new = {}
+    for k, v in d.items():
+        if isinstance(v, (np.ndarray, pd.Series)):
+            d_new[k] = tuple(v.tolist())
+        elif isinstance(v, list):
+            d_new[k] = tuple(v)
+        elif isinstance(v, dict):
+            d_new[k] = immutify_dictionary(v)
+        else:
+            if hasattr(v, "dtype"):
+                d_new[k] = v.item()
+            else:
+                d_new[k] = v
+
+    return dict(sorted(d_new.items(), key=lambda item: item[0]))
+
+
+def hash_dictionary(d):
+    d_hashable = immutify_dictionary(d)
```

Comment:
Is this step necessary? I would expect you could only hash the json string and be good.